packages/playground/website/src/components/ensure-playground-site/ensure-playground-site-is-selected.tsx

```diff
@@ -82,13 +84,23 @@ export function EnsurePlaygroundSiteIsSelected({
 				return;
 			}
 
+			// If only the 'modal' parameter changes in searchParams, don't reload the page
+			const notRefreshingParam = 'modal';
+			const oldParams = new URLSearchParams(prevUrl?.search);
+			const newParams = new URLSearchParams(url?.search);
+			oldParams.delete(notRefreshingParam);
+			newParams.delete(notRefreshingParam);
+			if (oldParams.toString() === newParams.toString()) {
```

Comment:
It turns out that this prevents us from automatically creating a temporary site when no other site is selected.
![Screenshot 2024-10-31 at 11 27 10 AM](https://github.com/user-attachments/assets/747c6c4e-e5d1-4076-948e-77868a5e15e8)

So it looks like, we need to add another condition:
Only avoid reloading the page when there is already an active Playground.

I suppose this makes sense. :)

Reply:
Would you mind if I just make that change and land this PR?

Reply:
@brandonpayton go ahead! Thanks :)

Reply:
Done! https://github.com/WordPress/wordpress-playground/pull/1908/commits/c06b09ba1e82da1dd2f628a9f50d759cbcea85f7

Once the tests pass, let's go ahead and merge this.